assignments/function-conversion.js

```diff
@@ -16,18 +16,33 @@ myFunction();
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {
+    return param;
+}
 
+anotherFunction("Example");
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => {
+    return param1 + param2;
+}
+
+add(1,2);
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+let subtract = (param1, param2) => {
+    return param1 - param2;
```

Comment:
Nice! You converted all the ES5 syntax functions to arrow functions! Try out the stretch too when you get a chance. 👍